### ISSUE TITLE
Preloader Application on first load and on View switches

### ISSUE THREAD

**Issue by rdaggett** [dmachi/dojox_application#201]
Hi there, I was wondering if there is a current way to add a loading message when switching views? I also noticed that the splash is deprecated, but I do need a pre-loader when the application is first loading. So having a pre-loader on first load and loading message when switching views would be nice to have included in this framework or is there a current workaround for this functionality?

Thanks

**Comment by rdaggett** [comment]
Closing this issue, I found that you can indeed use a pre-loader in transitioning views.

**Comment by edchat** [comment]
silverelf can you share how you were able to work around this?  
Thanks

**Comment by rdaggett** [comment]
What I did was I customized the dojox/layout/ContentPane pre-loading message by overwriting the css that is attached to that and loading my pages through the ContentPane. I am not sure if there is a different or more efficient way of doing this. But I am still working on other ways of implementing this into my desktop application. 

But it would be nice to have that integrated into the framework. I believe there was a splash screen which no longer supported. But I am currently looking the "transit" : "my/app/transit" in the dojox/app framework.
